llvm/lib/Target/RISCV/MCTargetDesc/RISCVELFStreamer.h

```diff
@@ -12,7 +12,7 @@
 #include "RISCVTargetStreamer.h"
 #include "llvm/MC/MCELFStreamer.h"
 
-using namespace llvm;
+namespace llvm {
```

Comment:
There's a `namespace llvm {` below that you should move instead.

Reply:
Right, that's why the build is failing. Let me make sure all the backends this is touching build locally for me before making it ready for review again.